src/sass/components/_button.scss

```diff
@@ -33,4 +33,13 @@
     background-color: #2a2a2a;
     color: #ffffff;
   }
+
+  &:active {
+    background-color: $primary;
+    border: 1px solid $primary;
+  } 
+  
+  &.active {
+    background-color: $primary;
```

Comment:
it'd be better if icon on active was white.
state for non-active: black icon, white background
hover for non-active: white icon, orange background
state for active: white icon, orange background
hover for active: black icon, white background
main menu works this way.